content-src/components/TopSites/TopSites.js

```diff
@@ -6,7 +6,7 @@ const TopSites = React.createClass({
     return (<section className="top-sites">
       <h3 className="section-title">Top Sites</h3>
       <div className="tiles-wrapper">
-        {props.sites.map((site) => (<a key={site.url} className="tile" href={site.url}>
+        {props.sites.map((site, idx) => (<a key={idx} className="tile" href={site.url}>
```

Comment:
Actually, using `idx` may not be good enough since using a numerical index may not be unique enough:

> The situation gets more complicated when the children are shuffled around (as in search results) or if new components are added onto the front of the list (as in streams). In these cases where the identity and state of each child must be maintained across render passes, you can uniquely identify each child by assigning it a `key`:
> &mdash; http://facebook.github.io/react/docs/multiple-components.html#dynamic-children